src/importchecker/importchecker.py

```diff
@@ -290,14 +290,14 @@ def getModulesImportingNameFrom(self, name, module):
         return result
 
 
-def main(path=None):
+def main(path=None, stdout=None):
     cwd = os.getcwd()
-    lencwd = len(cwd)+1
+    lencwd = len(cwd) + 1
 
     try:
         path = path or sys.argv[1]
     except IndexError:
-        print "No path supplied"
+        print("No path supplied", file=stdout)
         sys.exit(1)
```

Comment:
FWIW you can `sys.exit("No path supplied")` and it will print the message to stderr and exit with status 1.